lib/vue-rails-form-builder/form_helper.rb

```diff
@@ -5,6 +5,7 @@ def vue_form_with(**options)
         raise "Your Rails does not implement form_with helper."
       end
 
+      options[:camelize] ||= false
```

Comment:
Note that `camelize` option is not passed to the `fields_for` unlike the `builder` option.

See:

https://github.com/rails/rails/blob/375a4143cf5caeb6159b338be824903edfd62836/actionview/lib/action_view/helpers/form_helper.rb#L1929-L1933